packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -135,14 +136,21 @@ export class GCFBootstrapper {
     };
   }
 
-  private async handleScheduled(id: string, req: express.Request) {
-    const body = (Buffer.isBuffer(req.body)
+  private async handleScheduled(id: string, req: express.Request, eventName: string) {
```

Comment:
Consider renaming this function since it now handles scheduled and PubSub events.